new/LIT7087QeneDaggazmacc.xml

```diff
@@ -0,0 +1,91 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7087QeneDaggazmacc" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ደጃዝማች፡ ውቤ፡ ኢታብዝህ፡ ኀዘነ፡ ልቡ<choice><sic>ነ</sic><corr resp="JK">ና</corr></choice>፡ 
+                    በሞተ፡ አሀዱ፡ ወልድከ፡ እንበለ፡ ጊዜሁ፡ <sic resp="JK">ወመዋዕሁ</sic>፤</title>
```

Comment:
It is ሉ / ወመዋዕሉ in the end of the verse.

Reply:
remove mark up in `title`